studio/stores/pgmeta/MetaStore.ts

```diff
@@ -1,6 +1,6 @@
 import Papa from 'papaparse'
 import { makeObservable, observable } from 'mobx'
-import { find, isUndefined, isEqual, isEmpty, chunk, maxBy } from 'lodash'
+import { find, isUndefined, isEqual, isEmpty, chunk, filter, map } from 'lodash'
```

Comment:
side note FE team - can we get away from using `find`, `filter`, `map`, `isUndefined` from lodash

these are all pretty basic with native JS

```js
[].find(x => XX)
[].filter(x => XX)
[].map(x => XX)
value === undefined
```